### ISSUE TITLE
Import Bank Statements

### ISSUE THREAD

**Issue by dcowan-london** [bigcapitalhq/bigcapital#521]
The current bank import is very limited. Are there any plans to expand it? 
The features I’m looking for are:

- Accepting other date formats - currently, it only accepts YYYY-MM-DD format
- In / Out columns - currently, the import expects all amounts to be in one column, but many bank exports have separate Deposit / Withdrawal columns

**Comment by abouolia** [comment]
Yes, these features in the development backlog, but you can easily (in Google Sheet and Excel) change the date format and aggregate the amounts in one column and then import it. yes understand there's extra steps but would as temporary solution until we ship them. now we work in developing the banking services like transactions matching and bank reconciliation.

**Comment by raghavJhaRV** [comment]
@elforjani13 @suhaibaffan @kochie @ANasouf @benpsnyder @cloudsbird @Champetaman Sorry for disturbing you guys but i wanted help regarding this feature i am trying to implement but i wasn't able to access documentation so I had to ask for your help.
